docs/guides/component-testing/getting-started.mdx

```diff
@@ -125,6 +125,11 @@ Let's assume the Stepper Component consists of two `buttons`, one used to **decr
 to **increment** it as well as a `span` tag that sits in the middle of the buttons to show the
 current value of the counter.
 
+:::info
+To see examples of the Stepper Component and its tests in each Framework, visit our [Component Testing Quick Start Apps Repo](https://github.com/cypress-io/component-testing-quickstart-apps)
+
+:::
+
```

Comment:
I think you need to add some specific steps to copy from https://github.com/cypress-io/component-testing-quickstart-apps/blob/main/react/my-awesome-app/src/components/Stepper.jsx otherwise when you get to where it says:

"Switch back to the browser you opened for testing, and you should now see the Stepper Component in the spec list. "

there are no specific instructions to make sure that the Stepper Component has been added.